ergo-nipopow/src/lib.rs

```diff
@@ -7,7 +7,7 @@
 #![deny(non_camel_case_types)]
 #![deny(non_snake_case)]
 #![deny(unused_mut)]
-#![deny(dead_code)]
+//#![deny(dead_code)]
```

Comment:
If you export types via `pub use ...` the warnings should go away.

Reply:
Thanks!